production/helm/loki/templates/tokengen/job-tokengen.yaml

```diff
@@ -60,6 +60,8 @@ spec:
               mountPath: /shared
             - name: config
               mountPath: /etc/loki/config
+            - name: runtime-config
+              mountPath: /var/{{ include "loki.name" . }}-runtime
```

Comment:
Same comments as for the `statefulset-write.yaml` file.

Reply:
done :)